### DISCUSSION TITLE
Extending v2 support

### DISCUSSION THREAD

**Discussion by weirdan** [spatie/array-to-xml#199] [General]
I'm a maintainer of `vimeo/psalm` and we would like to to use `spatie/array-to-xml` to replace one of our dependencies that has become unmaintained. The thing is, we need to support PHP 7.4 for our v5 release cycle (a year or two, I imagine).

`spatie/array-to-xml:2.16.0` fits the bill just fine, but there are some fixes we would like to backport from v3 (to support PHP 8.1 and beyond).

I'd be happy to offer my help to support v2 branch for as long as we use it, which may benefit other people too. Of course, there's always an option to have our own fork, but I would rather avoid it if it's possible.

@freekmurze would you be open to this?

Refs: vimeo/psalm#8852

**Comment by freekmurze** [comment]
Hi @weirdan I'm open to this. Feel free to send PRs to the v2 branch 👍
